### PR TITLE
Multi language support

### PR DESCRIPTION
**Pull request by abtheo**
Adds i18n language framework and converts all strings to translation keys.

### CODE REVIEW

File: src/views/Dashboard/TreasuryDashboard.tsx [hunk 1]
```diff
@@ -105,8 +133,12 @@ function TreasuryDashboard() {
       <div className="hero">
         <Box component="div" color="text.primary">
           <p>Wen (3,3) becomes (🦦,🦦)</p>
-          <h1>Welcome to Otter Kingdom</h1>
-          <h3>The Decentralized Reserve Memecoin</h3>
+          <h1>
+            <Trans i18nKey="dashboard.otterKingdom" />
```
Comment (otterclamking) [inline src/views/Dashboard/TreasuryDashboard.tsx]: There are mixing use of `Trans` component and `t`. Feel we can unify these with `t`.

**Review by otterclamking — COMMENTED** [review]
Need some top margin on mobile.

deploy-preview-13--otterclam netlify app_(iPhone 6_7_8)

**Comment by otterclamking** [comment]
Hi, a PR https://github.com/OtterClam/otter-frontend/pull/12 merged. Need to resolve the conflicts and add new strings from that. 

I feel we could hide the language switch button and merge this first, so others can work on the version with i18n. After we have a new language, we show the language switch button.

File: src/views/Dashboard/TreasuryDashboard.tsx (hunk 1 above)
Comment (abtheo) [reply src/views/Dashboard/TreasuryDashboard.tsx]: Added padding on mobile

image

**Comment by abtheo** [comment]
Resolved the conflicts from latest merge, still need to add the NFT page to the translations but honestly we might be able to skip that for right now, because there is no Language button on the NFT page (yet)

I agree we should get this merged ASAP so people can work with i18n, otherwise this PR is gonna be a real pain to maintain. I can open a new PR to migrate everything from `Trans` to `t()` in a couple days.

**Pull request merged**